docs/upgrading_renku.rst

```diff
@@ -0,0 +1,62 @@
+.. upgrading_renku:
+
+Upgrading a Renku Project
+=========================
+
+The Renku Project is in constant flux! This means that often there are changes
+to RenkuLab (e.g. the web platform (UI) and the knowledge graph (KG)), and ``renku``
+(the command-line interface (CLI), which is installed via ``pipx`` in the
+interactive environments).
+
+Using new features on RenkuLab
+------------------------------
+
+New features in the UI and KG on RenkuLab are announced in the :ref:`release_notes`.
+
+When we update components of the RenkuLab platform, usually you will have access
+to these new features without doing anything special. E.g. when we added the ability
+for anonymous (i.e. not logged in) users to launch interactive environments, all
+projects were immediately accessible to anonymous users without the project owner
+needing to change any settings.
+
+.. _renku_cli_upgrade:
+
+Using new ``renku`` CLI features
+--------------------------------
+
+Note: This section is for upgrading the version of ``renku`` CLI installed into
```

Comment:
If you want this to appear as a special "note" box, you can use the `note` directive - see https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions